configs/_base_/datasets/mot_challenge.py

```diff
@@ -0,0 +1,88 @@
+# dataset settings
+dataset_type = 'MOTChallengeDataset'
+data_root = 'data/MOT17/'
+
+# data pipeline
+train_pipeline = [
+    dict(
+        type='UniformSample',
+        num_ref_imgs=1,
+        frame_range=10,
+        filter_key_img=True),
+    dict(
+        type='TransformBroadcaster',
+        share_random_params=True,
+        transforms=[
+            dict(type='LoadImageFromFile'),
+            dict(type='LoadTrackAnnotations', with_instance_id=True),
+            dict(
+                type='RandomResize',
+                scale=(1088, 1088),
+                ratio_range=(0.8, 1.2),
+                keep_ratio=True,
+                clip_object_border=False),
+            dict(type='PhotoMetricDistortion')
+        ]),
+    dict(
+        type='TransformBroadcaster',
+        share_random_params=False,
```

Comment:
Is there a basis for setting this parameter to true or false? Suggest to add a comment